test/manual/atlas-data-lake-testing/atlas_data_lake_testing.prose.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+const { expect } = require('chai');
+const { withClient } = require('../../integration/shared');
+
+// TODO: These tests are not fully implemented per the spec
+describe('Atlas Data Lake - prose', function () {
+  it(
+    '1. Test that the driver properly constructs and issues a killCursors command to Atlas Data Lake.',
+    /**
+     * For this test, configure an APM listener on a client and execute a query on the test.driverdata collection
+     * that will leave a cursor open on the server (e.g. specify batchSize=2 for a query that would match 3+ documents).
+     * Drivers MAY iterate the cursor if necessary to execute the initial find command but MUST NOT iterate further to avoid executing a getMore.
+     *
+     * Observe the CommandSucceededEvent event for the find command and extract the cursor's ID and namespace from the response document's cursor.id
+     * and cursor.ns fields, respectively. Destroy the cursor object and observe a CommandStartedEvent and CommandSucceededEvent for the killCursors command.
+     *
+     * Assert that the cursor ID and target namespace in the outgoing command match the values from the find command's CommandSucceededEvent.
+     * When matching the namespace, note that the killCursors field will contain the collection name and the database may be inferred from either
+     * the $db field or accessed via the CommandStartedEvent directly.
+     *
+     * Finally, assert that the killCursors CommandSucceededEvent indicates that the expected cursor was killed in the cursorsKilled field.
+     *
+     * Note: this test assumes that drivers only issue a killCursors command internally when destroying a cursor that may still exist on the server.
+     * If a driver constructs and issues killCursors commands in other ways (e.g. public API), this test MUST be adapted to test all such code paths.
```

Comment:
We are clearly not implementing the full description of the case correctly here, I think maybe this is testing one of the "other ways"? But in that case we still need to test the first part - the internal and the public codepaths, I assume. Could use help interpreting this line.

Reply:
Yea I'm not seeing the logic to issue a query to have the outstanding cursor remain open, and then subsequently issue killCursors. I'm not sure but our cursor.close() may issue a killCursors, so we need to test doing it this way with a db.command and see what happens with a cursor.close call as well.